app/src/main/java/com/keylesspalace/tusky/components/account/list/ListsForAccountFragment.kt

```diff
@@ -49,7 +53,11 @@ class ListsForAccountFragment : DialogFragment(), Injectable {
     lateinit var viewModelFactory: ViewModelFactory
 
     private val viewModel: ListsForAccountViewModel by viewModels { viewModelFactory }
-    private val binding by viewBinding(FragmentListsForAccountBinding::bind)
+
+    private var _binding: FragmentListsForAccountBinding? = null
+
+    // This property is only valid between onCreateDialog and onDestroyView
+    private val binding get() = _binding!!
```

Comment:
Move the binding completely into `onCreateDialog`. It is not needed anywhere else.

Reply:
load() does still use it.

(That ties in with my observation / todo above for using a LoadState above.)